src/PFVariables.hpp

```diff
@@ -26,6 +22,16 @@ class PFVariables {
         Cabana::Grid::Experimental::Impl::FFTBackendDefault>>
         fft_calculator;
 
+    // string to be used for Cabana's BOVWriter's prefix argument
+    std::string save_prefix(const int index, std::string run_name) {
+        std::stringstream name;
+#ifdef RESULTS_PATH
```

Comment:
What specifically are you asking to be removed?  If it's RESULTS_PATH, I believe that is needed to specify the folder where results are output; otherwise the results get dumped wherever the console is when the executable is executed.  This would make one have to cd to the desired results folder and then run the executable from there (and I don't even know how that would work with a Frontier batched job).
Changing it to CABANAPF_RESULTS_PATH is fine with me.

Reply:
Well I would generally assume that it will be saved in the current directory, but that does get annoying sometimes. You could pass the path on the command line, but that's pretty far outside the scope of this PR. Let's just rename `CABANAPF_RESULTS_PATH` for now